### PR TITLE
feat: remove CTA

### PR DESCRIPTION
**Pull request by asadali145**
### Relevant Tickets
https://github.com/openedx/frontend-app-ora-grading/issues/360

### Description:

The CTA was added two years ago. It is enabled by default, and the notification is always visible. This PR removes the CTA. If we still need it, we can put it behind a flag.

### Screen Shots:
Before:
Screenshot 2024-08-20 at 1 26 05 PM

After:
Screenshot 2024-10-02 at 12 33 20 PM

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @asadali145!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/openedx-unmaintained`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by asadali145** [comment]
@jansenk Would you be able to review this PR?

**Comment by asadali145** [comment]
@brian-smith-tcril Would you be able to review this PR?

**Comment by brian-smith-tcril** [comment]
Looking at this PR raises a few questions for me:
* Do we want to remove the CTA?
* Do we want to completely remove the notification banner if “preferences center” would be a broken link?
  * This effectively turns `ACCOUNT_SETTINGS_URL` into a "show/hide" notification banner toggle which seems less than ideal.
* Do we need the notification banner *at all*?

My feeling is both the CTA and notification banner should just be removed completely. This would require going through [the deprecation process](https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0021-proc-deprecation.html).

Would fully removing both the notification banner and the CTA solve the problem you're trying to solve with this PR?

**Comment by asadali145** [comment]
> Would fully removing both the notification banner and the CTA solve the problem you're trying to solve with this PR?

@brian-smith-tcril Yes, removing both solves our problem. We don't want to display the banners all the time.

Thanks for reviewing the PR. Do you want me to start the depreciation process for these banners?

**Comment by brian-smith-tcril** [comment]
> Do you want me to start the depreciation process for these banners?

That would be wonderful @asadali145! Thank you!

**Comment by asadali145** [comment]
@brian-smith-tcril I have created this DEPR ticket https://github.com/openedx/frontend-app-ora-grading/issues/360. Do you have any suggestions?

[EDIT]: I also [posted](https://openedx.slack.com/archives/C02R2PF91QA/p1727784048391039) the question in the Open edX Slack to confirm the usage of the notifications banner.

File: src/containers/NotificationsBanner/index.jsx [hunk 1]
```diff
@@ -6,7 +7,8 @@ import { PageBanner, Hyperlink } from '@openedx/paragon';
 
 import messages from './messages';
 
-export const NotificationsBanner = () => (
+// eslint-disable-next-line no-confusing-arrow
```
Comment (awais-ansari) [inline src/containers/NotificationsBanner/index.jsx]: Please follow the eslint recommendations. Other than PR LGTM.

**Review by awais-ansari — CHANGES_REQUESTED** [review]

**Review by brian-smith-tcril — CHANGES_REQUESTED** [review]
I still think the best path forward for this would be to deprecate and fully remove both the CTA and notifications banner. If that isn't the direction we end up going I still do not approve of the way this is implemented.

* I don't believe using `ACCOUNT_SETTINGS_URL` to determine if the notifications should appear or not is a good pattern. One other solution that comes to mind would be showing a different string without a link when `ACCOUNT_SETTINGS_URL` is not set.

Also, now that there seems to be more discussion around how we want to handle this, I believe it would be best to split this PR up. There should be one for removing the CTA, and one for modifying the notifications banner. That will allow the discussions to be targeted, and blockers for one won't get in the way of the other.

**Comment by asadali145** [comment]
Thanks @brian-smith-tcril and @awais-ansari. I have updated this PR to remove the CTA. I'll create a new PR to update the notifications banner. I have updated the DEPR ticket as well.

**Comment by asadali145** [comment]
@brian-smith-tcril Just a reminder that this PR needs review along with https://github.com/openedx/frontend-app-ora-grading/pull/362 as discussed to create separate PRs.

**Review by brian-smith-tcril — APPROVED** [review]
Let's merge it! (context on DEPR approval https://github.com/openedx/frontend-app-ora-grading/issues/360#issuecomment-2452307704)

**Comment by asadali145** [comment]
@brian-smith-tcril Thanks for approval. I do not have merge access. Please merge it if you can OR I can ask someone else to merge this.

**Comment by asadali145** [comment]
@brian-smith-tcril I have rebased this one as well.

**Comment by brian-smith-tcril** [comment]
@asadali145 sorry I missed this! It looks like it needs another rebase. I've communicated my intention to merge, so as soon as it is rebased I'll hit the button.

**Comment by asadali145** [comment]
@brian-smith-tcril This is rebased. You can merge.

**Pull request merged**